src/PowerShell/Microsoft.WinGet.DSC/Microsoft.WinGet.DSC.psm1

```diff
@@ -446,6 +452,78 @@ class WinGetPackageManager
     }
 }
 
+[DSCResource()]
+class WinGetConfigRoot
+{
+    [DscProperty(Key)]
+    [string]$WinGetConfigRoot = "WinGetConfigRoot"
+
+    [DscProperty()]
+    [string]$Path = ""

```

Comment:
There should be validation that `Exist: $true` cannot be used with an empty path in `Set`, otherwise that results in a state conflict where the user has requested a `Exist` to be true, but the result of the setter will always result in `Exist` being false